tests/fixtures/inference.py

```
@@ -314,3 +318,77 @@ def patched_dsc_garak_kfp(admin_client) -> Generator[DataScienceCluster]:
+        client=admin_client,
+        name="vllm-runtime-cpu-fp16",
+        namespace=model_namespace.name,
+        template_name=RuntimeTemplates.VLLM_CUDA,
+        deployment_type=KServeDeploymentType.RAW_DEPLOYMENT,
+        runtime_image="quay.io/rh-aiservices-bu/vllm-cpu-openai-ubi9"
+        "@sha256:ada6b3ba98829eb81ae4f89364d9b431c0222671eafb9a04aa16f31628536af2",
+        containers={
+            "kserve-container": {
+                "args": ["--port=8032", "--model=/mnt/models", "--served-model-name={{.Name}}"],
+                "ports": [{"containerPort": 8032, "protocol": "TCP"}],
+                "volumeMounts": [{"mountPath": "/dev/shm", "name": "shm"}],
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/dev/shm"

(S108)